TSC-Charter.md

```diff
@@ -36,16 +36,27 @@ approved by the Board.
 
 ## Section 4. Establishment of the TSC.
 
-TSC memberships are not time-limited. There is no maximum size of the TSC.
-The size is expected to vary in order to ensure adequate coverage of important
-areas of expertise, balanced with the ability to make decisions efficiently.
-The TSC must have at least three members.
+TSC memberships are for one year terms that must be recertified. Motions for
+recertification are automatic and require a vote. Members are recertified if a
+simple majority of TSC members vote in favor of recertifying the individual as
+a TSC member. Such members will continue through the next year's term.
+Individuals failing recertification may return to the TSC after no less than
+six months following normal TSC motion.
+
+There is no maximum size of the TSC. The size is expected to vary in order to
+ensure adequate coverage of important areas of expertise, balanced with the
+ability to make decisions efficiently. The TSC must have at least three members.
+
+Individuals nominated to the TSC must be members of at least one Node.js
+Working Group at the time of their nomination and must maintain active
+participation in at least one Working Group throughout their term.
```

Comment:
This one concerns me a bit.  For some areas we have teams as opposed to WGs (ex crypto) and I hate to lose those voices from the TSC.  We should look at how many of the current TSC members would be affected by this requirement as I think it would b quite a high number.